### PR TITLE
Net cleanup

### PR DESCRIPTION
**Pull request by ghartman17022**

### CODE REVIEW

File: debian/cuttlefish-common.prerm [hunk 1]
```diff
@@ -1,7 +1 @@
-# Automatically added by dh_installinit
-if [ -x "/etc/init.d/cuttlefish-common" ] || [ -e "/etc/init/cuttlefish-common.conf" ]; then
-	invoke-rc.d cuttlefish-common stop || exit $?
-fi
-# End automatically added section
-
```
Comment (jemoreira357) [inline debian/cuttlefish-common.prerm]: is this no longer needed?
was it needed before?

File: debian/changelog [hunk 2]
```diff
@@ -2,9 +2,10 @@ cuttlefish-common (0.9.8) UNRELEASED; urgency=medium
 
   * Changes to enable arm builds
   * Fix dependencies for buster
-  * Add qemu-user-static on non-amd64
```
Comment (zenczykowski) [inline debian/changelog]: this removal seems wrong?

Comment (zenczykowski) [reply debian/changelog]: and I'm not convinced about the add in the next line either... did someone previously misgenerate this or???

File: debian/cuttlefish-common.init [hunk 3]
```diff
@@ -83,7 +44,8 @@ create_interface() {
     /sbin/brctl addbr "${bridge}"
     /sbin/brctl stp "${bridge}" off
     /sbin/brctl setfd "${bridge}" 0
-    /sbin/ifconfig "${bridge}" "${gateway}" netmask "${netmask}" up
+    ip -4 addr add ${gateway}/30 broadcast + dev "${bridge}"
```
Comment (zenczykowski) [inline debian/cuttlefish-common.init]: "${gateway}/30"

File: debian/cuttlefish-common.init [hunk 4]
```diff
@@ -83,7 +44,8 @@ create_interface() {
...
+    ip link set dev ${bridge} up
```
Comment (zenczykowski) [inline debian/cuttlefish-common.init]: "${bridge}"

File: debian/cuttlefish-common.init [hunk 5]
```diff
@@ -100,7 +62,7 @@ create_interface() {
     --dhcp-no-override
 
     ip tuntap add dev "${tap}" mode tap group cvdnetwork
```
Comment (zenczykowski) [inline debian/cuttlefish-common.init]: suggest moving tap creation before dnsmasq gets launched

File: debian/cuttlefish-common.init [hunk 6]
```diff
@@ -114,7 +76,7 @@ destroy_interface() {
     tap="$(printf cvd-${1}tap-%02d $2)"
```
Comment (zenczykowski) [inline debian/cuttlefish-common.init]: consistency says $2 should be "${2}"
and "cvd-${1}tap-%02d"

(google shell style says always "${}" wrap var expansions if you can, although $1 is actually an exception to the ${} wrapping requirement, but not to "" wrapping)

File: debian/cuttlefish-common.udev [hunk 7]
```diff
@@ -1 +1,2 @@
 ACTION=="add", KERNEL=="vhost-vsock", SUBSYSTEM=="misc", MODE="0660", GROUP="cvdnetwork"
+ACTION=="add", KERNEL=="vhost-net", SUBSYSTEM=="misc", MODE="0660", GROUP="cvdnetwork"
```
Comment (zenczykowski) [inline debian/cuttlefish-common.udev]: file should probably sorted in same order as module loading

**Comment by adelva1984** [comment]
Closing this review as all of the feedback was integrated into my change, which just landed.

**Pull request closed without merging**